source/includes/steps-install-shell-linux-rpm.yaml

```diff
@@ -0,0 +1,21 @@
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: navigate-dlc
+ref: navigate-dlc-linux-rpm
+---
+title: "Download the Red Hat 64-bit ``.rpm`` package."
```

Comment:
Minor: DL center lists this download entry as: 

> Redhat 64-bit

I could swear they changed that on us at some point!